src/PayPal/Order/PayPalOrderPresenter.php

```diff
@@ -95,10 +97,7 @@ public function getTotalAmountFormatted()
             return '';
         }
 
-        return Tools::displayPrice(
-            (float) $this->paypalOrderDataProvider->getTotalAmount(),
-            Currency::getCurrencyInstance(Currency::getIdByIsoCode($this->paypalOrderDataProvider->getCurrencyCode()))
-        );
+        return \Context::getContext()->getCurrentLocale()->formatPrice((float) $this->paypalOrderDataProvider->getTotalAmount(), $this->paypalOrderDataProvider->getCurrencyCode());
```

Comment:
```suggestion
        return \Context::getContext()->getCurrentLocale()->formatPrice((float) $this->paypalOrderDataProvider->getTotalAmount(), $this->paypalOrderDataProvider->getCurrencyCode());
```

I think this could be simplified with `Tools::getContextLocale`